src/ManagedProcess.php

```diff
@@ -26,9 +26,10 @@ public function __construct( array $steps, ?int $timeout = 60 ) {
 	 * @param ProcessManager $manager
 	 * @param array|null $data
 	 * @param string|null $pid ID of the exsting process to continue
+	 * @param string[] $addInfo Additional information can be passed inside the array or [] if none
 	 * @return string ProcessID
 	 */
-	public function start( ProcessManager $manager, $data = [], $pid = null ) {
+	public function start( ProcessManager $manager, $data = [], $pid = null, $addInfo = [] ) {
```

Comment:
It is not actually "information" that is being passed, but command line arguments for the `maintenance/processExecution.php` script